### PR TITLE
Authenticated/Unauthenticated user redirection

### PR DESCRIPTION
**Pull request by mihaibudeci**

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -16,7 +16,10 @@ const router = new Router({
     {
       path: '/login',
       name: 'login',
-      component: () => import('./views/user/Login.vue')
+      component: () => import('./views/user/Login.vue'),
+      meta: {
```
Comment (mihaibudeci) [inline src/router.js]: Trebuie sters

File: src/router.js [hunk 2]
```diff
@@ -31,7 +34,10 @@ const router = new Router({
     {
       path: '/register',
       name: 'register',
-      component: () => import('./views/user/Register.vue')
+      component: () => import('./views/user/Register.vue'),
+      meta: {
```
Comment (mihaibudeci) [inline src/router.js]: Trebuie sters

File: src/views/user/Login.vue [hunk 3]
```diff
@@ -4,13 +4,15 @@
       <b-row align-h="center" class="mt-5">
         <b-col cols="12" md="6">
           <b-card class="p-3">
-            <h3 class="mb-4">Log In</h3>
+            <h2 class="mb-4 text-center">Log In</h2>
+            <div style="text-align:center">
+              <a href="/register">or create an account</a>
+            </div>
             <b-form @submit.prevent="login" v-if="show" :novalidate="true">
```
Comment (spirlici) [inline src/views/user/Login.vue]: here `v-if="show"` is not needed, you can just remove it and also remove property `show: true` from data.

File: src/views/user/Login.vue [hunk 4]
```diff
@@ -83,7 +84,7 @@ export default {
     login () {
       authService.login(this.form, 'error')
```
Comment (spirlici) [inline src/views/user/Login.vue]: Delete the second parameter, `'error'`. It was just used for testing.
Search in the whole project and if it is called with two arguments just delete the second one.

**Pull request merged**